ctapipe/ml/sklearn.py

```diff
@@ -140,6 +140,7 @@ class Classifier(Model):
     ).tag(config=True)
 
     invalid_class = Integer(-1).tag(config=True)
+    positive_class = Integer(default_value=1).tag(config=True)
```

Comment:
@LukasNickel here

Reply:
Wouldn't default value 0 be more reasonable?

Reply:
No, normally you use labels = [0, 1] with one being the positive class.

Reply:
True.
It is just annoying, that the simtel id for gammas is actually 0 :(